stable/mysql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mysql
-version: 0.3.7
+version: 0.3.8
```

Comment:
Perhaps this justifies a "minor" rather than a "patch" version bump?

Reply:
Yeah, this should be minor.